Frends.AzureEventHub.Receive/Frends.AzureEventHub.Receive/Definitions/Enums.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Frends.AzureEventHub.Receive.Definitions;
+
+/// <summary>
+/// Authentication methods.
+/// </summary>
+public enum AuthenticationMethod
+{
+    /// <summary>
+    /// Connection string.
+    /// </summary>
+    ConnectionString,
+
+    /// <summary>
+    /// SAS Token.
```

Comment:
I was thinking about some more information on what is SAS Token. E.g. https://learn.microsoft.com/en-us/azure/cognitive-services/translator/document-translation/how-to-guides/create-sas-tokens?tabs=Containers